src/velero/chart/values.yaml

```diff
@@ -0,0 +1,6 @@
+storage:
+  internal:
+    enabled: false
+    remoteSelector: {}
+    remoteNamespace: ""
+  cidr: ""
```

Comment:
I think it would be helpful to call this something that makes it more clear it is used for network policy generation.